CHANGES.md

```diff
@@ -3857,16 +3857,15 @@ With the release of GAP 4.5, we also encourage more users to take
 advantage of the increasingly mature binary distributions which are now
 available. These include:
 
-  - The binary `rsync` distribution for GAP on Linux PCs with i686
-    or x86\_64 compatible processors provided by Frank Lübeck, see
-    <http://www.math.rwth-aachen.de/~Frank.Luebeck/gap/rsync>.
+  - The binary [`rsync` distribution](http://www.math.rwth-aachen.de/~Frank.Luebeck/gap/rsync)
+    for GAP on Linux PCs with i686 or x86_64 compatible processors provided by Frank Lübeck.
 
-  - **BOB**, a tool for Linux and Mac OS X to download and build GAP
-    and its packages from source provided by M. Neunhöffer:
-    <http://www-groups.mcs.st-and.ac.uk/~neunhoef/Computer/Software/Gap/bob.html>.
+  - [**BOB**](http://www-groups.mcs.st-and.ac.uk/~neunhoef/Computer/Software/Gap/bob.html),
```

Comment:
BOB was decommissioned, and the URL no longer works. Fine to mention is as a history, but remove the URL

Reply:
there are more historical, no longer working, URLs there. I'm not sure they should be removed.

Reply:
not sure too, but for BOB in particular perhaps you can put a note like "Update (2020): BOB is not currently supported, the archived repository is located at https://github.com/gap-system/bob".

Reply:
one other place I noticed was location of homalg repo.